src/common/SPELLS_DEATH_KNIGHT.js

```diff
@@ -173,6 +173,26 @@ export default {
     icon: 'spell_shadow_deadofnight',
   },
 
+  //Blood Tier Sets
+  //T20 2 Piece
+  BLOOD_T20_2PC_GRAVEWARDEN: {
+    id: 242001,
+    name: 'Death Knight T20 Blood 2P Bonus',
```

Comment:
The preferred spell naming scheme for set piece definitions is `T20 4 set bonus`. The spec name can be omitted in the context of a spec analysis.

Reply:
Fixed